src/marqo/s2_inference/model_registry.py

```diff
@@ -364,6 +364,8 @@ def _get_open_clip_properties() -> Dict:
 
         # New models as of Marqo 2.7.0
         # Added for: Open CLIP 2.24.0
+        # TODO the open_clip/ViT-SO400M-14-SigLIP-384/webli is not available on torch 1.12
+        # Add this model back when we upgrade the torch version
```

Comment:
So we kept this model, right? The comment's a bit confusing

Reply:
Updated